go/state/state_db.go

```diff
@@ -192,6 +194,9 @@ type stateDB struct {
 	// A transaction local cache of storage values to avoid double-fetches and support rollbacks.
 	data *common.FastMap[slotId, *slotValue]
 
+	// Transient storage
+	transientData *common.FastMap[slotId, common.Value]
```

Comment:
I would name the variable `transientStorage`

Reply:
Also it wold be nice to define a new private type for this storage like

`type transientStorage common.FastMap[slotId, common.Value]`